src/styles/Login.css.js

```diff
@@ -0,0 +1,49 @@
+import { css } from 'react-emotion';

```

Comment:
I've never used react-emotion but I'm down to give it a try. Why do you like it? Have you used other CSS in JS libraries?

Reply:
I suspected as much, that's why I threw those stlyes in a separate folder. If I were to use those styles in the Login component itself, the way JS scoping works prevents you from potential naming conflicts with other components within the application because all of your style definitions are scoped to the Component itself.

Mostly just makes component reuse and maintenance a little easier. Emotion is cool because you just get to write regular CSS while doing all that